File: backend/src/main/kotlin/ch/streckeisen/mycv/backend/cv/project/ProjectUpdateDto.kt
package ch.streckeisen.mycv.backend.cv.project

import java.time.LocalDate

data class ProjectUpdateDto(
    val id: Long?,
    val name: String?,
    val role: String?,
    val description: String?,
    val projectStart: LocalDate?,
    val projectEnd: LocalDate?,
    val links: List<ProjectLinkUpdateDto>?
)

File: frontend/src/types/ProjectUpdateDto.ts
import { ProjectLinkUpdateDto } from '@/types/ProjectLinkUpdateDto.ts'

export interface ProjectUpdateDto {
  id?: number;
  name?: string;
  role?: string;
  description?: string;
  projectStart?: string;
  projectEnd?: string;
  links?: ProjectLinkUpdateDto[];
}
